DB/src/main/java/io/deephaven/db/v2/NestedPartitionedDiskBackedTable.java

```diff
@@ -34,7 +34,7 @@ public NestedPartitionedDiskBackedTable(TableDefinition tableDefinition,
                                             Set<String> internalPartitions) {
         //noinspection unchecked
         super(checkTableDefinitionRequirements(tableDefinition),
-                NestedPartitionedDiskBackedTable.class.getSimpleName() + '[' + tableDefinition.getNamespace() + ',' + tableDefinition.getName() + ']',
+                NestedPartitionedDiskBackedTable.class.getSimpleName(),
```

Comment:
Would be better to replace the removed fragment with:
` + '[' + tableLocationProvider + ']'`